source/2020-07-24-ember-3-20-released.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Ember 3.20 Released
+author: Jen Weber
+tags: Releases, 2020, 3, 3.20, Version 3.x
+responsive: true
+---
+
+Today the Ember project is releasing version 3.20 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.21 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.20
+
+Ember.js 3.20 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is COUNT (#) new feature, COUNT (#) deprecations, and COUNT (#) bugfixes in this version.
+
+#### New Features (2)
+
+First new feature (1 of 2)
+
+Second new feature (2 of 2)
+
+#### Deprecations (0)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.20, please review the [Ember.js 3.20.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.20.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.20
+
+#### New Features (0)
+
+No new features introduced in Ember Data 3.20.
+
+#### Deprecations (0)
+
+No new deprecations introduced in Ember Data 3.20.
+
+For more details on changes in Ember Data 3.20, please review the
+[Ember Data 3.20.0 release page](https://github.com/emberjs/data/releases/tag/v3.20.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+<!--alex ignore easy-->
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npm install -g ember-cli-update
+ember-cli-update
```

Comment:
Can we suggest:

```suggestion
npx ember-cli-update
```

Instead of `npm i -g`?

Reply:
Done!